spectrum-offchain-lm/src/backlog_stream.rs

```diff
@@ -0,0 +1,61 @@
+use std::sync::Arc;
+
+use futures::{Stream, StreamExt};
+use log::trace;
+use spectrum_offchain::{
+    backlog::Backlog,
+    data::order::{OrderUpdate, PendingOrder},
+};
+use tokio::sync::Mutex;
+
+use crate::{
+    bundle::BundleRepo,
+    data::{
+        order::{Order, OrderProto},
+        AsBox, BundleId,
+    },
+};
+
+/// Create backlog stream that drives processing of order events.
+pub fn backlog_stream<'a, S, TBacklog, TBundles>(
```

Comment:
Doesn't this conflict with `backlog_stream` from `spectrum-offchain/src/backlog/process.rs`?
We can do one thing at a time, then this method will only enrich orders `OrderProto => Order` and then be composed with `backlog_stream`.

Reply:
I actually moved the file from `spectrum-offchain/src/backlog/process.rs` to `spectrum-offchain-lm/src/backlog_stream.rs` entirely, so no conflict. I did it because the `BundleRepo` trait resides in `spectrum-offchain-lm`. Is that ok?

Reply:
Backlog implementation doesn't depend on LM. This conflict arose because you tried to mix two different kinds of responsibility in one method. If you leave backlog_stream.rs in `spectrum-offchain` as is and only implement `OrderProto => Order` in `spectrum-offchain-lm`, then compose them everything will be ok.

Reply:
Done.